README.md

```diff
@@ -140,3 +140,29 @@ CoreDNS is a fast and flexible DNS server. It has a focus of service discovery i
 -	Recommended Skills: Golang, Microsoft Azure
 -	Mentor(s): Yong Tang (@yongtang)
 
+### TiKV
+
+TiKV is an open-source distributed transactional Key-Value database. [https://tikv.org](https://tikv.org).
+
+#### Optimize Raft in cross datacenter scenarios
+
+- Description: The original Raft consensus algorithm is not optimized for cross data center scenarios, where we cannot avoid high latency between two data centers but alleviate it. The goal for this section is to introduce some optimizations from Paxos that help us enhance Raft.
+- Recommended Skills: Rust, Raft
+- Mentor(s): Ana Hobden (@hoverbear), JianJun Li (@busyjay), Nick Cameron (@nrc)
+- Issue: https://github.com/pingcap/raft-rs/issues/178, https://github.com/pingcap/raft-rs/issues/145, https://github.com/pingcap/raft-rs/issues/136, https://github.com/pingcap/raft-rs/issues/135
+
+
+#### Speed up TiKV compilation
+
+- Description: Compiling Rust programs may take a long time, especially for TiKV. We usually have to wait at least 20+ minutes. The goal for this section is to reduce Rust compiling time by diving into Rust compiler components such as Cargo and linking.
```

Comment:
Similar question - this looks like it will need changes to cargo and not tikv?

IMHO if this includes changes _only_ for cargo and not tikv, this should go under the Mozilla org instead of the CNCF org.

Reply:
@nikhita 

I will update the description, mostly we want to do the optimization for TiKV itself, not for Rust.